packages/parser/src/validators/spec/swagger.ts

```diff
@@ -113,25 +111,27 @@ function validatePathParameters(params: OpenAPIV2.ParameterObject[], pathId: str
     .filter(param => param.in === 'path')
     .forEach(param => {
       if (param.required !== true) {
-        throw ono.syntax(
-          'Validation failed. Path parameters cannot be optional. ' +
-            `Set required=true for the "${param.name}" parameter at ${operationId}`,
+        throw new SyntaxError(
+          `Validation failed. Path parameters cannot be optional. Set \`required=true\` for the \`${param.name}\` parameter at \`${operationId}\`.`,
         );
       }
 
       const match = placeholders.indexOf(`{${param.name}}`);
       if (match === -1) {
-        throw ono.syntax(
-          `Validation failed. ${operationId} has a path parameter named "${param.name}", ` +
-            `but there is no corresponding {${param.name}} in the path string`,
+        throw new SyntaxError(
+          `Validation failed. \`${operationId}\` has a path parameter named \`${param.name}\`, but there is no corresponding \`{${param.name}}\` in the path string.`,
         );
       }
 
       placeholders.splice(match, 1);
     });
 
   if (placeholders.length > 0) {
-    throw ono.syntax(`Validation failed. ${operationId} is missing path parameter(s) for ${placeholders}`);
+    const list = new Intl.ListFormat('en', { style: 'long', type: 'conjunction' }).format(
```

Comment:
`Intl` seems so so handy for stuff like this

Reply:
big `Intl` head over here